psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -177,6 +177,19 @@ def scf_iterate(self, e_conv=None, d_conv=None):
         if (self.iteration_ == 0) and self.reset_occ_:
             self.reset_occupation()
 
+        upcm = 0.0
+        if core.get_option('SCF', 'PCM'):
+            calc_type = core.PCM.CalcType.Total
+            if core.get_option("PCM", "PCM_SCF_TYPE") is "SEPARATE":
```

Comment:
`==`, not `is` here. Latter is for singletons `None`, `True`, `False`, I believe.

Reply:
Fixed.